### ISSUE TITLE
Docstring needs to be fixed in twisted.words

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6479]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Upasana reported|
|-|-|
|Trac ID|trac#6479|
|Type|enhancement|
|Created|2013-04-26 19:24:22Z|

************* Module twisted.words
W9201:  1,0: The opening/closing of docstring should be on a line by themselves

Attachments:

* [bug_38280.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e5d/e5d5db2513a4ccfb08c5f5d94f73a0948664daea/c1d13de859dd3a880e789215234141e93ce479ce.patch) (411 bytes) - added by Upasana on 2013-04-26 19:28:11Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6479 6479
type__enhancement enhancement
reporter__Upasana Upasana
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__easy easy
time__1367004262000000 1367004262000000
changetime__1367080423000000 1367080423000000
version__None None
owner__thijs thijs
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Upasana removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38284]) Apply bug_38280.patch: Correct the opening/closing of the twisted.words module docstring.

Author: Upasana
Reviewer: thijs
Fixes: [#6479](https://github.com/twisted/twisted/issues/6479)
